src/ctapipe/image/extractor.py

```diff
@@ -1531,15 +1531,15 @@ def adaptive_centroid(waveforms, peak_index, rel_descend_limit, centroids):
         sum_ += waveforms[j]
         jsum += j * waveforms[j]
         j -= 1
-        if waveforms[j] > peak_amplitude:
+        if j > 0 and waveforms[j] > peak_amplitude:
```

Comment:
Should be >=

Reply:
Yes, you're right

Reply:
That was more of a note to self that I realized after I opened the PR, but thanks for fixing :+1: